server/plugins/DelayUGens.cpp

```diff
@@ -5681,6 +5681,14 @@ void PitchShift_Ctor(PitchShift *unit)
 	pchratio = ZIN0(2);
 	winsize = ZIN0(1);
 
+	// If the window size is <= 2 samples, scsynth will freeze (presumably some
```

Comment:
You should add a FIXME / TODO to this, since the original problem is still not fully understood

Reply:
I tested this and it works fine; please just add a FIXME and I'm happy to merge! I'm also going to add a note about fixmes/todos to the style guide